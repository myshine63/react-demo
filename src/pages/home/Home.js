import React from "react";

function Home() {
  return (
    <h1>hello he pan</h1>
  )
}


export default React.memo(Home)