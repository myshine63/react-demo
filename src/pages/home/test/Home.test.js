import React from "react";
import renderer from 'react-test-renderer'
import Home from "../Home";


it('渲染home页面', () => {
  const home = renderer.create(<Home/>).toJSON();
  expect(home).toMatchSnapshot();
})