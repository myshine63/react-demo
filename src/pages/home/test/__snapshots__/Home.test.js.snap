// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`渲染home页面 1`] = `
<h1>
  hello he pan
</h1>
`;
